source/falaise/snemo/datamodels/sim_digi_data.h

```diff
@@ -24,60 +24,84 @@
+			/// The collection of calorimeter digi hits
+			typedef std::vector<sim_calo_digi_hit> calo_digi_hit_collection_type;
 
-  /// Constructor
-  sim_digi_data();
+			/// The collection of tracker digi hits
+			typedef std::vector<sim_tracker_digi_hit> tracker_digi_hit_collection_type;
 
-  /// Destructor
-  virtual ~sim_digi_data();
+			/// The collection of trigger info
+			typedef std::vector<sim_trigger_digi_data> trigger_info_data_collection_type;
 
-  /// Check validity
-  bool is_valid() const;
+			/// Constructor
+			sim_digi_data();
 
-  /// Reset
-  void reset();
+			/// Destructor
+			virtual ~sim_digi_data();
 
-  /// Return the const collection of trigger digi events
-  const sim_digi_event_collection_type& get_digi_events() const;
+			/// Check validity
+			bool is_valid() const;
 
-  /// Return the mutable collection of trigger digi events
-  sim_digi_event_collection_type& grab_digi_events();
+			/// Reset
+			void reset();
 
-  /// Return the const container of auxiliary properties
-  const datatools::properties& get_auxiliaries() const;
+			/// Return the const collection of trigger digi events
+			const calo_digi_hit_collection_type & get_calo_digi_hits() const;
 
-  /// Return the mutable container of auxiliary properties
-  datatools::properties& grab_auxiliaries();
+			/// Return the mutable collection of trigger digi events
+			calo_digi_hit_collection_type & grab_calo_digi_hits();
 
-  /// Smart print
-  virtual void tree_dump(std::ostream& a_out = std::clog, const std::string& a_title = "",
-                         const std::string& a_indent = "", bool a_inherit = false) const;
+			/// Return the const collection of trigger digi events
+			const tracker_digi_hit_collection_type & get_tracker_digi_hits() const;
 
- private:
-  datatools::properties _auxiliaries_;  //!< Auxiliary properties
+			/// Return the mutable collection of trigger digi events
+			tracker_digi_hit_collection_type & grab_tracker_digi_hits();
 
-  sim_digi_event_collection_type
-      _digi_events_;  //!< Simulated digitized events (trigger + readout datas)
+			/// Return the const collection of trigger digi events
+			const trigger_info_data_collection_type & get_trigger_digi_data() const;
 
-  DATATOOLS_SERIALIZATION_DECLARATION()
-};
+			/// Return the mutable collection of trigger digi events
+			trigger_info_data_collection_type & grab_trigger_digi_data();
 
-}  // end of namespace datamodel
+			/// Return the const container of auxiliary properties
+			const datatools::properties & get_auxiliaries() const;
+
+			/// Return the mutable container of auxiliary properties
+			datatools::properties & grab_auxiliaries();
+
+			/// Smart print
+			virtual void tree_dump(std::ostream& a_out = std::clog,
+														 const std::string& a_title = "",
+														 const std::string& a_indent = "",
+														 bool a_inherit = false) const;
+
+		private:
+
+			datatools::properties _auxiliaries_;  //!< Auxiliary properties
```

Comment:
Please explain the purpose and use cases of this data member. What data does it represent?